### PR TITLE
L2-330: Update neuron delay UI

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

UI for Dissolve Delay Modal after creating a neuron.

# Changes

* new SedDissolveDelay component.
* Add SetDissolveDelay step to CreateNeuronModal component.

# Tests

* Updated CreateNeuronModal tests to check for "Set Dissolve Delay" screen after creating neuron.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
Screenshot

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis @peterpeterparker please review

File: frontend/svelte/src/lib/modals/neurons/CreateNeuronModal.svelte [hunk 1]
```diff
@@ -54,7 +65,8 @@
   showBackButton={currentStep === Steps.StakeNeuron}
   on:nnsBack={goBack}
 >
-  <span slot="title">{$i18n.neurons?.[titleKey]}</span>
+  <span class:hello-world={false} slot="title">{$i18n.neurons?.[titleKey]}</span
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/modals/neurons/CreateNeuronModal.svelte:68]: Is `class:hello-world={false}` something for future?

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte [hunk 2]
```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+
+  import Card from "../../components/ui/Card.svelte";
+  import { i18n } from "../../stores/i18n";
+  import { secondsToDuration } from "../../utils/date.utils";
+  let EIGHT_YEARS = 60 * 60 * 24 * 365 * 8 + 60 * 60 * 24 * 2; // Extra two days for two leap years
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:7]: What do you think about having some app level constants for SECONDS_IN_DAY, SECONDS_IN_YEAR etc?

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    </div>
+  </Card>
+  <div class="buttons">
+    <button on:click={skip} class="secondary full-width"
+      >{$i18n.neurons.skip}</button
+    >
+    <button
+      class="primary full-width"
+      disabled={disableUpdate}
+      data-test="update-button">{$i18n.neurons.update_delay}</button
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:76]: 👍

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+    .details {
+      margin-top: var(--padding);
+      display: flex;
+      justify-content: space-around;
+    }
+  }
+
+  input[type="range"] {
+    -webkit-appearance: none;
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:108]: I guess with our browser matrix those prefixes are not necessary https://caniuse.com/?search=appearance

**Review by mstrasinskis — CHANGES_REQUESTED** [review]
LGTM. Just a couple of comments.

File: frontend/svelte/src/lib/modals/neurons/CreateNeuronModal.svelte (hunk 1 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/modals/neurons/CreateNeuronModal.svelte:68]: oops, haha, I was trying things

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte (hunk 2 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:7]: could be useful, I can add them to constants.

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte (hunk 4 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:108]: good point

File: frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte (hunk 3 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/modals/neurons/SetDissolveDelay.svelte:77]: I wanted to propose and decided to first give it a try.

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis ready for another review

File: frontend/svelte/src/lib/constants/constants.ts [hunk 5]
```diff
@@ -3,3 +3,8 @@ export const LIST_PAGINATION_LIMIT: number = 100;
  * The infinite scroll observe an element that finds place after x % of last page.
  */
 export const INFINITE_SCROLL_OFFSET: number = 0.2;
+export const SECONDS_IN_MINUTE = 60;
+export const SECONDS_IN_HOUR = SECONDS_IN_MINUTE * 60;
+export const SECONDS_IN_DAY = SECONDS_IN_HOUR * 24;
+// Taking into account 1/4 of leap year
+export const SECONDS_IN_YEAR = ((4 * 365 + 1) * SECONDS_IN_DAY) / 4;
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/constants/constants.ts:10]: Nice 👍

**Review by mstrasinskis — APPROVED** [review]
LGTM. Thanks for constants

**Pull request merged**